nilearn/datasets/atlas.py

```diff
@@ -1961,17 +1961,18 @@ def fetch_atlas_talairach(level_name, data_dir=None, verbose=1):
     return Bunch(maps=atlas_img, labels=labels, description=description)
 
 
+@rename_parameters(replacement_params={"version": "atlas_type"})
```

Comment:
I haven't set `end_version`. Shall I update it with a specific version?

Reply:
yeah we usually have a 2 releases deprecation cycle so it would be 0.13.1